packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -212,15 +213,25 @@ class StatefulGraphChatClient implements StatefulClient<GraphChatClient> {
     Providers.globalProvider.onActiveAccountChanged(this.onActiveAccountChanged);
     this._eventEmitter = new ThreadEventEmitter();
     this.registerEventListeners();
-    this._notificationClient = new GraphNotificationClient(this._eventEmitter, graph('mgt-chat', GraphConfig.version));
+    this._notificationClient = new GraphNotificationClient(this._eventEmitter, this._graph);
```

Comment:
This is not a good change as we would lose the telemetry telling us how much usage this component gets.